docs/modules/ROOT/pages/index.adoc

```diff
@@ -43,8 +43,8 @@ All Defender components, where applicable, provide support for the following cha
 
 - *Ethereum Mainnet* and its testnets *Rinkeby*, *Ropsten*, *Kovan*, and *Goerli*.
 - https://docs.matic.network/docs/develop/network-details/network[*Polygon* (Matic)] and *Mumbai*.
-- https://arbitrum.io/[*Arbitrum*] and *Arbitrum Rinkeby*.
-- https://optimism.io/[*Optimism*] and *Optimism Kovan*.
+- https://arbitrum.io/[*Arbitrum*], *Arbitrum Rinkeby*, and *Arbitrum Goerli*.
+- https://optimism.io/[*Optimism*], *Optimism Kovan*, and *Optimism Goerli*.
```

Comment:
should we make a note that Arbitrum Rinkeby and Optimism Kovan are planned for deprecation?should we make a note that Arbitrum Rinkeby and Optimism Kovan are planned for deprecation?

Reply:
Might be, but I don't want to make a public announcement yet.
I'd like to hear opinions from @spalladino.

Otherwise I'd prefer to merge right away and then chat about showing the deprecation notice later